test/test_ops.py

```diff
@@ -630,7 +630,7 @@ def test_nms_ref(self, iou, seed):
         boxes, scores = self._create_tensors_with_iou(1000, iou)
         keep_ref = self._reference_nms(boxes, scores, iou)
         keep = ops.nms(boxes, scores, iou)
-        assert torch.allclose(keep, keep_ref), err_msg.format(iou)
+        torch.testing.assert_close(keep, keep_ref, msg=err_msg.format(iou))
```

Comment:
Point 2. from above does not apply here, since supplying a `msg: str` overwrites the default diagnostic message.